app/assets/javascript/application.js

```diff
@@ -34,7 +34,7 @@ Sentry.init({
   release: window.sentryConfig.release,
   integrations: [],
   tracesSampleRate: 0, // Disable tracing (performance monitoring, doesn't impact errors)
-  ignoreErrors: [/'Object\.prototype\.hasOwnProperty\.call\(o,"telephone"\)'/],
+  ignoreErrors: [/'Object\.prototype\.hasOwnProperty\.call\([eo],"telephone"\)'/],
```

Comment:
Now the message comes with `e, "telephone"` instad of `o, "telephone"`. It will ignore both notations now.